certs/certs.go

```diff
@@ -0,0 +1,67 @@
+// Package certs handles persisting and loading certs
+package certs
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"errors"
+	"fmt"
+	"github.com/levenlabs/locksmith/config"
+	"log"
+	"os"
+	"time"
+)
+
+func init() {
+	if config.CertsFile == "" {
+		return
+	}
+	// make sure we can write to the file
+	if _, err := os.Stat(config.CertsFile); err != nil {
```

Comment:
Instead of checking if the file exists, you might as well just do a Create and Chmod on it either way and save some code.